### PR TITLE
benchmark: change a parameter level from global to session

### PR DESCRIPTION
**Pull request by glkappe**
set global tidb_opt_distinct_agg_push_down = 1
> 1105 - Variable 'tidb_opt_distinct_agg_push_down' is a SESSION variable and can't be used with SET GLOBAL
> 时间: 0.028s

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by glkappe** [comment]
希望增加更详细的描述：

https://asktug.com/t/topic/35114/6?u=%E6%88%B7%E5%8F%A3%E8%88%9F%E4%BA%A2
1. 用户如何执行下载的 sql
2. 打印解决样式和详解。

File: benchmark/v4.0-performance-benchmarking-with-tpch.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ learner_config.log-level: "info"
 
 ```sql
 set global tidb_allow_batch_cop = 1;
-set global tidb_opt_distinct_agg_push_down = 1;
+set session tidb_opt_distinct_agg_push_down = 1;
```
Comment (zz-jason) [inline benchmark/v4.0-performance-benchmarking-with-tpch.md:95]: Please change the section name "全局变量配置" to "变量配置". Also, please note that all the following tests should be finished in the same database connection.

Comment (TomShawn) [reply benchmark/v4.0-performance-benchmarking-with-tpch.md:95]: PTAL again

Comment (zz-jason) [reply benchmark/v4.0-performance-benchmarking-with-tpch.md:94]: 在文档的开头加个注意事项吧，因为设置了 session variable，所有 query 都需要在当前这个 session 中跑

File: benchmark/v4.0-performance-benchmarking-with-tpch.md [hunk 2]
```diff
@@ -85,13 +85,17 @@ logger.level: "info"
 learner_config.log-level: "info"
 ```
 
-##### 全局变量配置
+##### 变量配置
+
+> **注意：**
+>
+> 部分参数为 SESSION VARIABLE，建议所有 QUERRY 都需要在当前 SESSION 中执行。
```
Comment (TomShawn) [inline benchmark/v4.0-performance-benchmarking-with-tpch.md:92]:
```suggestion
> 部分参数为 SESSION 变量。建议所有查询都在当前 SESSION 中执行。
```

Comment (zz-jason) [reply benchmark/v4.0-performance-benchmarking-with-tpch.md:92]: the suggestion loos good to me, @glkappe please address this comment.

**Pull request merged**